frontend/README.md

```diff
@@ -0,0 +1,41 @@
+# Measure Repository Service Frontend
+
+A frontend application for a prototype implementation of a [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html).
+
+- Installation(#installation)
+
+  - [Prerequisites](#prerequisites)
+  - [Local Installation](#local-installation)
+
+- Usage(#usage)
+
+## Installation
+
+### Prerequisites
+
+- [Node.js >=11.15.0](https://nodejs.org/en/)
```

Comment:
In reality this is probably not true. We should list at least node 16 here since the codebase is in npm 8+